contrib/runners/mistral_v2/mistral_v2.py

```diff
@@ -170,7 +170,10 @@ def _construct_workflow_execution_options(self):
         # into raw block. If there is any jinja expressions, Mistral will try to evaulate
         # the expression. If there is a local context reference, the evaluation will fail
         # because the local context reference is out of scope.
-        for k, v in six.iteritems(parent_context.get('chain', {}).get('params', {})):
+        chain_ctx = parent_context.get('chain') or {}
```

Comment:
Lol. by default, dict.get will return None if the key doesn't exist.  so with `or {}`, the variable will be assigned {} anyway.  Am i missing something?